templates/JSResolverOCHTTPSTemplate.js

```diff
@@ -745,7 +744,8 @@ function resolveGrapgDBqueryForGraphQLMutation (obj, querySchemaInfo) {
         if (obj.definitions[0].selectionSet.selections[0].selectionSet != undefined) {        
             returnBlock = returnStringOnly(obj.definitions[0].selectionSet.selections[0].selectionSet.selections, querySchemaInfo);
         }
-        let ocQuery = `CREATE (${nodeName}:${querySchemaInfo.returnTypeAlias} {${inputFields.fields}})\nRETURN ${returnBlock}`;        
+        let ocQuery = '';
+        ocQuery = `CREATE (${nodeName}:\`${querySchemaInfo.returnTypeAlias}\` {${inputFields.fields}})\nRETURN ${returnBlock}`;
```

Comment:
Can these two lines be merged into one? Any reason to declare as empty first?

Reply:
Also, it should be `const`